events/tpcreaction.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    name: 'messageCreate',
+    once: false,
+    execute (message) {
+  if (message.thread?.name.startsWith('SCREENSHOT CONTEST') || message.channel.id === '830210202464813056') {

```

Comment:
In general, we want to keep away from as many logical, static checks as we can. Hardcoding values like this can prove to be problematic for maintence purposes.

Reply:
Is there a better way to make sure the channel is right without hardcoding?

Reply:
@michaelcmelton ^